science/proxmark3-iceman/Portfile

```diff
@@ -12,7 +12,7 @@ checksums           sha256  36006cf30c30a533027de851fe9a96593a093e2e03cddf1703aa
                     rmd160  dba82dee74f69e829b0a786ab7b56b8086524fad \
                     size    24755014
 
-revision            0
+revision            1
```

Comment:
it looks like this change will not affect the files that are being installed - correct? In that case there is no need to increase the `revision` field. I do notice that the package doesn't build on many of the buildbots; it this change is to resolve the build failure then you can just add the `PATH` without increasing the revision.

However, shouldn't you depend on a port to have something installed in `${prefix}/libexec/gnubin`?